src/PolygonClipper/SegmentComparer.cs

```diff
@@ -33,97 +33,97 @@ public int Compare(SweepEvent? x, SweepEvent? y)
             return 1;
         }
 
-        SweepEvent se_old_l, se_new_l;
-        Func<bool, int> less_if;
+        SweepEvent perhapsInversedX, perhapsInversedY;
+        Func<bool, int> lessIf;
```

Comment:
You could call the method directly here given that it is static. I would apply aggressive inlining also though.